sdk/keyvault/Azure.Security.KeyVault.Keys/Readme.md

```diff
@@ -148,7 +148,7 @@ Key key = client.CreateKey("key-name", KeyType.EllipticCurve);
 // You can specify additional application-specific metadata in the form of tags.
 key.Tags["foo"] = "updated tag";
 
-KeyBase updatedKey = client.UpdateKey(key);
+KeyBase updatedKey = client.UpdateKey(key, key.KeyMaterial.KeyOps);
```

Comment:
This isn't a comment on the docs but KeyOps should be optional on update key.

Reply:
Issue https://github.com/Azure/azure-sdk-for-net/issues/6932 created